.github/ISSUE_TEMPLATE/cherry_pick_template.md

```diff
@@ -1,25 +1,29 @@
 ---
-name: Cherry-pick template
+name: Cherry-pick request
 about: Used to request a cherry-pick. See go.amp.dev/cherry-picks
 title: "\U0001F338 Cherry-pick request for #<ISSUE_NUMBER> into #<RELEASE_ISSUE> (Pending)"
-labels: 'Type: Release'
+labels:
```

Comment:
Super-nit: We use the hyphenated "cherry-pick" in most of the documentation, including the `name` field above.

Reply:
I like this nit! Updated labels and fixed template.